desktop/electron/videoGen.ts

```diff
@@ -0,0 +1,34 @@
+import { downloadFile } from "./reportProcessing";
+import { getProjectPath } from "./metadata";
+import fs from "fs";
+
+export async function imageToVideo(imagePath: string, fps: number=7, videoLength: number=10): Promise<string> { 
+
+    const endpoint = 'https://iguana.alexo.uk/v9/video';
+
+    console.log("Image Path: ", imagePath);
+    const form = new FormData();
+
+    // const filePromise = new Promise((resolve, reject) => {
+    //     fs.readFile(imagePath, (err, data) => {
+    //       if (err) { reject(err); }
+    //       resolve(dataurl.convert({ data, mimetype: 'image/png' }));
+    //     });
+    // });
+    // const imageUrl = await filePromise as string;
+
+    const imageBuffer = fs.readFileSync(imagePath);
+    const imageBlob = new Blob([imageBuffer], { type: 'image/png' });
```

Comment:
Is there  a reason for using PNG here, the images generated from Open Journey are JPEGS

Reply:
Sorry was just using png to test local images. I can change it to so that we can accept any image mime type.